openedx/adg/lms/applications/models.py

```diff
@@ -0,0 +1,122 @@
+    )
+    cover_letter = models.TextField(blank=True, )
+
+    class Meta:
+        app_label = 'applications'
+        verbose_name = _('User Application')
+        verbose_name_plural = _('User Applications')
+
+    def __str__(self):
+        return 'UserApplication {id}, for user {email}'.format(id=self.id, email=self.user.email)
+
+
+class Education(TimeStampedModel):
+    """
+    Model for user education qualification for application submission.
+    """
+    HIGH_SCHOOL_DIPLOMA = 'HD'
+    ASSOCIATE_DEGREE = 'AD'
+    BACHELOR_DEGREE = 'BD'
+    MASTERS_DEGREE = 'MD'
+    DOCTORAL_DEGREE = 'DD'
+
+    DEGREE_TYPES = [
+        (HIGH_SCHOOL_DIPLOMA, _('High School Diploma or GED')),
+        (ASSOCIATE_DEGREE, _('Associate Degree')),
+        (BACHELOR_DEGREE, _('Bachelor’s Degree')),
+        (MASTERS_DEGREE, _('Master’s Degree')),
+        (DOCTORAL_DEGREE, _('Doctoral Degree')),
+    ]
+
+    month_choices = month_choices()
+    year_choices = year_choices()
+
+    user_application = models.ForeignKey(
+        'UserApplication', related_name='education_set', related_query_name="education", on_delete=models.CASCADE,
+    )
+    name_of_school = models.CharField(verbose_name=_('Name of School / University'), max_length=255, )
+    degree = models.CharField(verbose_name=_('Degree Received'), choices=DEGREE_TYPES, max_length=2, )
+    ares_of_study = models.CharField(verbose_name=_('Area of Study'), max_length=255, blank=True)
+    date_started_month = models.CharField(choices=month_choices, max_length=2, )
+    date_started_year = models.IntegerField(choices=year_choices, )
+    date_completed_month = models.CharField(choices=month_choices, blank=True, max_length=2, )
+    date_completed_year = models.IntegerField(choices=year_choices, blank=True, null=True, )
+    is_in_progress = models.BooleanField(verbose_name=_('In Progress'), default=False, )
+
+    class Meta:
+        app_label = 'applications'
+
+    def __str__(self):
+        return 'Education {id}, for {degree}'.format(id=self.id, degree=self.degree)
+
+
+class WorkExperience(TimeStampedModel):
+    """
+    Model for user work experience for application submission.
+    """
+    month_choices = month_choices()
+    year_choices = year_choices()
+
+    user_application = models.ForeignKey(
+        'UserApplication', related_name='work_experiences', related_query_name="work_experience",
+        on_delete=models.CASCADE,
+    )
+    name_of_organization = models.CharField(verbose_name=_('Name of Organization'), max_length=255, )
+    job_position_title = models.CharField(verbose_name=_('Job Position / Title'), max_length=255, )
+    date_started_month = models.CharField(choices=month_choices, max_length=2, )
+    date_started_year = models.IntegerField(choices=year_choices, )
+    date_completed_month = models.CharField(choices=month_choices, blank=True, max_length=2, )
+    date_completed_year = models.IntegerField(choices=year_choices, blank=True, null=True, )
```

Comment:
I am not sure the criteria we are following here to add a verbose name or not, but if are adding so that they could be used in forms, we should add for these fields as well in translateable string.